getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -21,12 +21,13 @@ styleguide.
 Code structure
 --------------
 
-Indentation
-~~~~~~~~~~~
-
-Indent type: Tabs *(editor default)*
+Formatting conventions
```

Comment:
That would lead to having the "Formatting Conventions" heading twice. Also, indentation is an accurate heading. The word format encompasses all the rules that affect the look of a document, like line returns, line length, etc.

Reply:
How does it lead to having the formatting conventions heading twice? I don't understand.

Reply:
Ah my bad, they're two documents, I think I need some rest. 😪